lib/camerata.rb

```diff
@@ -129,6 +129,29 @@ def console(service, *args)
     end
     map rc: :console
 
+    desc "set_env TARGET_NS SOURCE_NS", "copy params from env to another"
+    def env_copy(target_ns, source_ns = "")
+      app_versions = Camerata::AppVersions.get_all source_ns
```

Comment:
![image](https://user-images.githubusercontent.com/21695512/91495228-4049dc00-e888-11ea-8985-8b3b48a8d993.png)

I would have expected variables like YUL_DEPLOY_SOLR_VERSION rather than YUL_DEPLOY_YUL_TEST_SOLR_VERSION